CliFx.Tests/Services/CommandInitializerTests.cs

```diff
@@ -69,7 +70,34 @@ private static IEnumerable<TestCaseData> GetTestCases_InitializeCommand()
                     new CommandOptionInput("i", new[] {"foo", "bar"}),
                     new CommandOptionInput("s", " ")
                 }),
-                new ConcatCommand {Inputs = new[] {"foo", "bar"}, Separator = " "}
+                new ConcatCommand { Inputs = new[] { "foo", "bar" }, Separator = " " }
+            );
+
+            //Will read a value from environment variables because none is supplied via CommandInput
+            yield return new TestCaseData(
+                new EnvironmentVariableCommand(),
+                GetCommandSchema(typeof(EnvironmentVariableCommand)),
+                new CommandInput(null, new CommandOptionInput[] { }),
+                new EnvironmentVariableCommand { Option = "A" }
```

Comment:
It would be nice to have a test that ensures that, when property type is not a collection, environment variable is always treated as single value.

Example:

Option of type `string`.
Environment variable: `value1,value2`
Resulting option value: `"value1,value2"`

Option of type `string[]`
Environment variable: `value1,value2`
Resulting option values: `[] {"value1", "value2"}`

Does this make sense?

Reply:
Got it, makes sense